src/coord/src/coord.rs

```diff
@@ -822,6 +849,42 @@ impl<S: Append + 'static> Coordinator<S> {
             .unwrap();
     }
 
+    async fn group_commit(&mut self) {
```

Comment:
Doc comments here would be great. I read the method, and .. it does a thing, but I couldn't tell you if it does the right thing. E.g. is it ok that writes to absent catalog entries don't complain (maybe!). Is `None` the right thing to say to each of the writes.